doc/rst/source/explain_symbols.rst_

```diff
@@ -306,14 +306,18 @@
         examples to use as a template.  The language allows you to design symbols that
         take many parameters and can make decisions based on these parameters.
 
-    **-Sk**\ *name*\ /\ *size*
-        **k**\ ustom symbol. We will look for a symbol definition file called *name*\ .def in
+    **-S**\ [**k**\ *name*\ [/\ *size*]]
+        A **k**\ ustom symbol. We will look for a symbol definition file called *name*\ .def in
         (1) the current directory, (2) in ~/.gmt or (3) in **$GMT_SHAREDIR**/custom.
-        The symbol defined in the definition file is of normalized unit size by default;
+        The symbol declared in the definition file is of normalized unit size by default;
```

Comment:
_defined_ is more correct than _declared_

Reply:
Yes, but I was trying to avoid defined in definition file.  It is very repetitive...

Reply:
COuld say "The symbol in the definition file is ..."

Reply:
OK, better because they are really being defined in those files.

Reply:
Done in master.